### PR TITLE
junit: support testSuite without testCases

### LINKED ISSUE #6: Error while reading junit file
Debug:
```bash
Run inception-health/otel-upload-test-artifact-action@v1
  
Verify Job<Unit Test> and Step<Run tests>
Trace Test file
##[debug]matchDirectories 'false'
##[debug]followSymbolicLinks 'true'
##[debug]implicitDescendants 'true'
##[debug]matchDirectories 'false'
##[debug]omitBrokenSymbolicLinks 'true'
##[debug]Search path '/home/runner/work/service-template/service-template/tests-junit.xml'
##[debug]Tracing file: /home/runner/work/service-template/service-template/tests-junit.xml
##[debug]Tracing TestSuites<Undefined>
##[debug]Tracing TestSuite<github.com/my-org/service-template/cmd/app>
##[debug]Traced 1 Files
Error: TypeError: Cannot read properties of undefined (reading 'map')
```

Junit Sample:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite tests="0" failures="0" time="0.000000" name="github.com/my-org/service-template/cmd/app" timestamp="2022-06-14T22:02:21Z">
		<properties>
			<property name="go.version" value="go1.18.3 linux/amd64"></property>
		</properties>
	</testsuite>
	<testsuite tests="0" failures="0" time="0.000000" name="github.com/my-org/service-template/cmd/app/docs" timestamp="2022-06-14T22:02:21Z">
		<properties>
			<property name="go.version" value="go1.18.3 linux/amd64"></property>
		</properties>
	</testsuite>
	<testsuite tests="3" failures="0" time="0.008000" name="github.com/my-org/service-template/internal/nodetemplate" timestamp="2022-06-14T22:02:21Z">
		<properties>
			<property name="go.version" value="go1.18.3 linux/amd64"></property>
		</properties>
		<testcase classname="github.com/my-org/service-template/internal/nodetemplate" name="Test_New/succesfully_creates_client" time="0.000000"></testcase>
		<testcase classname="github.com/my-org/service-template/internal/nodetemplate" name="Test_New/unsuccesfully_creates_client_with_missing_host" time="0.000000"></testcase>
		<testcase classname="github.com/my-org/service-template/internal/nodetemplate" name="Test_New" time="0.000000"></testcase>
	</testsuite>
	<testsuite tests="0" failures="0" time="0.000000" name="github.com/my-org/service-template/internal/nodetemplate/models" timestamp="2022-06-14T22:02:21Z">
		<properties>
			<property name="go.version" value="go1.18.3 linux/amd64"></property>
		</properties>
	</testsuite>
</testsuites>
```

### PR DESCRIPTION
**Pull request by v1v**
Closes https://github.com/inception-health/otel-upload-test-artifact-action/issues/6

### What

Allow testSuite without testCases

### Why

`JUnit` allows `testSuite` without any `testCase` see https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd#L18-L24

### Test

It worked in my test environment

image

And If try with a more complex JUnit -> https://raw.githubusercontent.com/mdelapenya/junit2otlp/main/TEST-sample2.xml

and it also worked

image

### CODE REVIEW

File: src/trace-junit.ts [hunk 1]
```diff
@@ -245,8 +247,12 @@ export function traceTestSuite({
       );
       return testCaseTimeSec;
     });
-    testCasesTimeSec =
-      testCasesTimeSec || testCasesTimes.reduce((r, i) => r + i, 0);
+
+    // If testSuite exists with testCases
+    if (testCasesTimes) {
+      testCasesTimeSec =
```
Comment (mdelapenya) [inline src/trace-junit.ts:253]: Careful with constant variables 😁 you probably need to declare it with `let`

Comment (v1v) [reply src/trace-junit.ts:253]: Uhm, It's defined in L231, is it also required in L253?

Comment (mdelapenya) [reply src/trace-junit.ts:253]: Ah ah sorry!! I missed the plurals 🤦‍♂️ in L236 it defines `const testCaseTimeSec`

**Review by ghost — APPROVED** [review]
Just tested it with New Relic OTLP. LGTM!

**Comment by v1v** [comment]
@nikordaris , I’m really impressed with this GitHub action. I’m really happy I could contribute back, 😊 let me know if this PR could be merged and released 🙏

**Review by nikordaris — APPROVED** [review]

**Comment by nikordaris** [comment]
Thanks for the contribution! I'm glad you are finding it helpful!

**Pull request merged**